.github/workflows/deploy.yml

```diff
@@ -21,17 +21,24 @@ jobs:
         id: install_fastn
         continue-on-error: false
         run: sh -c "$(curl -fsSL https://raw.githubusercontent.com/ftd-lang/fastn/main/install.sh)"
+      - name: Get latest commit information
+        id: get_commit
+        run: echo "::set-output name=commit_sha::$(git rev-parse HEAD)"
+      - name: Get raw zipball URL
+        id: get_zip_url
+        run: |
+          echo "::set-output name=zip_url::https://github.com/${{ github.repository }}/archive/${{ steps.get_commit.outputs.commit_sha }}.zip"
       - name: Build the pages with fastn
         id: build
         continue-on-error: false
         run: |
           echo "Using '$CI_REPOSITORY_NAME_SLUG/' as the base while building"
-          fastn build --base=/$CI_REPOSITORY_NAME/ 
+          fastn build --base=/$CI_REPOSITORY_NAME/ --zip-url=${{ steps.get_zip_url.outputs.zip_url }}
```

Comment:
Nopes. Have you done this to all repos? Goal is to not have to ask everyone who have used our templates to create our repo to have to update their workflow file. There are 3K plus people on discord, you think it is alright for you @harshdoesdev  to ask all of them to update their repo, or is it better you make a 5 line change in our Rust code and handle Github Pages internally?